storybookjs/storybook
# Fix semi broken __docgenInfo integration in addon info

Issue:
I'd like to see the component description from __docgenInfo in the story documentation in the info plugin.
I'd like to get the prop type from __docgenInfo for when the type comes back as "other".
PropVal had a problem where 'content' was being directly rendered as an object.

## What I did
added to PropTable
added to PropVal
added to Story


## How to test
Make a story with a component that has documented (via docgen-style text) description, and the description will show up.

Like this:
```
/** Component description here */
const Button = ({children, onClick}) => (
  <button onClick={onClick}>{children}</button>
)
Button.propTypes = {
  onClick: PropTypes.func // make sure this does not come from React, make it come from prop-types module.
}

storiesOf('Button', module)
  .addWithInfo('demo', 'Button with children', () => (
    <Button onClick={action('clicked')}>David</Button>
  ))
 .addWithInfo('demo', '', () => (
    <Button onClick={action('clicked')}>David</Button>
  ))
```

You will see a description under the info description, you will also see that the prop type is not 'other' when you use PropTypes from something other than React (for support of React >= 15.5.x.
